src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -95,6 +95,40 @@
                  version))))
     version))
 
+(defn level->int
+  "Returns an integer which represents the given status level.
+   The ordering of levels is :critical < :info < :debug."
+  [level]
+  (case level
+    :critical 0
+    :info 1
+    :debug 2))
+
+(defn apply-to-int-levels
```

Comment:
Is there something other than comparing that you would likely use this for though? If we can't come up with a better name, I would go with `compare-levels` over `apply-to-int-levels`, because I very hard time parsing the latter and understanding what it means.